### PR TITLE
donut 3 sec changes

### PR DESCRIPTION
**Pull request by EyeSpider**
This change adds a solitary cell just below the main area in sec, as well as improving the main area to host a cozy 3x3 table for all your various data disks and other items. Interrogation has been moved down a tiny bit to host for the new cell. It is equipped with an automatic locker, a bed, a mounted flash and a computer all for the prisoner's comfort.
.
This is needed because sometimes mixing prisoners isn't a good idea. Whether someone be particularly hostile, dangerous, etc. and need that extra eye kept on them, or if there are simply too many people to all cram into genpop, a solitary cell is very nice to have available. Also, a bigger table has been needed in donut3 sec for a while due to the volume of trash security like to throw onto the existing 2 tables which are normally used.

### CODE REVIEW

**Comment by Zonespace27** [comment]
Is it possible you could upload a photo of the map changes here?

**Comment by Kubius** [comment]
Map changes are automatically visualized by MapDiffBot.

https://github.com/goonstation/goonstation/pull/5436/checks?check_run_id=3032365376

**Comment by UrsulaMejor** [comment]
@Tenryumi

**Comment by EyeSpider** [comment]
I have fixed the APC issue. The new cell is now zoned as part of sec and the old zoning is deleted.

**Comment by EyeSpider** [comment]
Everything *should* be perfect now regarding any errors with the map or code itself.

**Comment by Tenryumi** [comment]
> 
> 
> @Tenryumi

Sorry for not responding sooner!
Now then...

Oh shiiit, I really like these changes! Merge that goodness!

**Comment by Tenryumi** [comment]
Wait
This removed the smiley face from the wiring here
image

You must understand
Yes, it's stupid
But it's important
Preserve the c=

**Comment by EyeSpider** [comment]
@Tenryumi Added back smiley face (didn't realise it was a face just thought it was weird wiring as I was doing some wiring of my own). Also *should* have resolved a merge conflict. I think.

**Comment by KB2222** [comment]
I'd love to see Donut 3 get a round table, and solitary brig cells. Glad to see this.

**Comment by Tenryumi** [comment]
Aight rad. I am now more than happy with this. These changes look really good!
Meeerge!

File: maps/donut3.dmm [hunk 1]
```diff
@@ -28748,19 +28915,18 @@
 /turf/simulated/floor/white,
 /area/station/medical/medbay/cloner)
 "igr" = (
-/obj/access_spawn/hos,
 /obj/machinery/door/airlock/pyro/glass/security{
-	aiControlDisabled = 1;
```
Comment (Sovexe) [inline maps/donut3.dmm:28753]: why was this removed?

**Review by Sovexe — CHANGES_REQUESTED** [review]

File: maps/donut3.dmm [hunk 2]
```diff
@@ -59245,21 +59475,15 @@
...
 /obj/cable{
-	d2 = 4;
-	icon_state = "0-4"
-	},
-/obj/machinery/light/emergency{
-	dir = 8;
-	pixel_x = -12
-	},
-/obj/machinery/sleeper/compact{
-	dir = 1
```
Comment (Sovexe) [inline maps/donut3.dmm:59258]: you also appear to be making undocumented changes to the sleepers

Comment (EyeSpider) [reply maps/donut3.dmm:59258]: I must've accidentally deleted the sleeper and replaced it with one which is identical (see images).
image
image
